torch/onnx/_internal/exporter/_registration.py

```diff
@@ -162,33 +159,19 @@ def from_torchlib(
                 target = _get_overload(qualified_name)
                 if target is None:
                     continue
-                for overload_func in aten_overloads_func.overloads:
-                    overload_func.signature = _schemas.OpSignature.from_function(
-                        overload_func,
-                        overload_func.function_ir.domain,
-                        overload_func.name,
-                    )
-                    onnx_decomposition = OnnxDecompMeta(
-                        onnx_function=overload_func,
-                        fx_target=target,
-                        is_custom=False,
-                        is_complex=False,
-                    )
-                    registry._register(target, onnx_decomposition)
-
-                for complex_func in aten_overloads_func.complex:
-                    overload_func.signature = _schemas.OpSignature.from_function(
-                        overload_func,
-                        overload_func.function_ir.domain,
-                        overload_func.name,
-                    )
-                    onnx_decomposition = OnnxDecompMeta(
-                        onnx_function=complex_func,
-                        fx_target=target,
-                        is_custom=False,
-                        is_complex=True,
-                    )
-                    registry._register(target, onnx_decomposition)
+
+                overload_func.signature = _schemas.OpSignature.from_function(
+                    overload_func,
+                    domain,
+                    name,
+                )
+                onnx_decomposition = OnnxDecompMeta(
+                    onnx_function=overload_func,
+                    fx_target=target,
+                    is_custom=False,
+                    is_complex=meta.is_complex,
+                )
+                registry._register(target, onnx_decomposition)
```

Comment:
Is there still registry api for custom ops?

Reply:
It just occurs to me that we might need to refactor ONNX tutorial on PyTorch/tutorial afer the migration.

Reply:
I agree. One thing is how to do want to expose the registration API which I don’t have a good idea yet